src/ui/link/index.tsx

```diff
@@ -72,8 +72,12 @@ const Link = forwardRef<HTMLButtonElement | HTMLAnchorElement, LinkProps>(
               )) ||
             ''
           : to;
+    const IS_ABSOLUTE_LINK_REGEX = /^((?:(http|https):\/\/)|\/\/)/;
```

Comment:
The code change itself looks good. However, I wonder if href should be prefixed.
For `to`, it makes sense because `to` must be one of routes. 
But for `href`, it's basically any URL. Should we prefix at the product side?

```
const { getBasePath } = useRouterActions();
const basePath = getBasePath();

const href = `${getBasePath}/something`;
```

Reply:
Is it too many places we need to prefix if it's done at the product level?